fossology/fossology
# feat(upload): make global decisions configurable in all upload pages

## Description

Make global decision configurable. from all upload pages.

### Changes

* add a function to store values in report_info page;

## How to test

* Upload a package and do some clearing with global decisions.
* Upload a some other version of package again and see if there are decisions applied automatically.
* Upload a some other version of package by clicking "Apply global decisions for selected upload" and check the global decisions in your package.
